assets/js/modules/analytics-4/components/widgets/index.js

```diff
@@ -26,3 +26,4 @@ export { default as TopCitiesWidget } from './TopCitiesWidget';
 export { default as TopCountriesWidget } from './TopCountriesWidget';
 export { default as TopTrafficSourceWidget } from './TopTrafficSourceWidget';
 export { default as TopConvertingTrafficSourceWidget } from './TopConvertingTrafficSourceWidget';
+export { default as ConnectGA4CTAWidget } from './ConnectGA4CTAWidget';
```

Comment:
Nit: please sort 😄 There's actually a related `ConnectGA4CTATileWidget` component already so we've found the right place 👍

Reply:
Updated, thank you!